### ISSUE TITLE
(Feature Request) Support for 1.19.4+ displayentities?

### ISSUE THREAD

**Issue by I0nTech** [elBukkit/MagicPlugin#1211]
https://www.reddit.com/r/Minecraft/comments/1223aco/the_new_item_display_entity_is_fun/

Would love an action in Magic which allows us to spawn and manipulate display entities, would greatly improve VFX for spells. If possible, some sort of DisplayEntityProjectile action would also be greatly appreciated if possible.

**Comment by PhvntxmX** [comment]
more like effectlib feature, not magic itself

**Comment by I0nTech** [comment]
Not exactly, displayentities aren't particles, which is what effectlib handles. Either way, I'd much prefer it as an action if it ever gets implemented, however all things considered the chances of that are slim enough that I'm going to end up doing it myself anyways.

**Comment by NathanWolf** [comment]
I will look into it- if there's Spigot API this should be easy. I would probably implement it as an EntityData flag (e.g. `display: true`) so you could use one in EntityProjectile, SpawnEntity, etc.

**Comment by NathanWolf** [comment]
I haven't had a chance to look into it at all- but if there are such things and they are easily accessible via API (or maybe NMS), then probably :)
